Client/multiplayer_sa/CMultiplayerSA.cpp

```diff
@@ -1498,6 +1498,29 @@ void CMultiplayerSA::InitHooks()
     MemPut((void*)0x6943AD, &fDuckingHealthThreshold);
     fDuckingHealthThreshold = 0;
 
+    // Lower the GTA shadows offset closer to ground/floor level
+    m_fShadowsOffset = 0.013f;            // GTA default = 0.06f
+    std::array<DWORD, 16> shadowAddr{
```

Comment:
Please move the array out of the function and mark it as static. Also, replace type `DWORD` with `uint32_t`.

Reply:
Thanks! This has been addressed.